clients/client/src/main/java/org/projectnessie/client/http/v1api/HttpUpdateNamespace.java

```diff
@@ -15,9 +15,9 @@
  */
 package org.projectnessie.client.http.v1api;
```

Comment:
is this naming pattern contradicting `org.projectnessie.api.v1.http` to some degree?

Reply:
It is, but I did not want to rename unnecessary packages :) In this PR I just wanted to make room for adding `api.v2`.

I can do the client renaming in a separate PR, if you think it might be worth the effort.